com.unity.render-pipelines.high-definition/Tests/Editor/HDRenderUtilitiesTests.cs

```diff
@@ -38,5 +38,6 @@ public void RenderThrowWhenTargetIsNotARenderTextureForTex2DRendering()
                 Texture2D.whiteTexture
             ));
         }
+
```

Comment:
Do you need this space? Did you add this file by mistake?

Reply:
It's a mistake. The test was originally in that file, so it looks like a mistake in reverting the file.